src/devices/QwiicButton/FlagsHelper.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
```

Comment:
we now use two line copyright header (remove third line in each file)

Reply:
Fixed

Reply:
@Esbob it doesn't seems fix for me. Are you sure you pushed the latest modficiations?

![image](https://user-images.githubusercontent.com/8145578/106171232-9c938d80-6191-11eb-850b-a91c07fefb03.png)